src/Batch/Client/changelog.md

```diff
@@ -1,5 +1,11 @@
 ## Azure.Batch release notes
 
+### Upcoming changes
+These changes are planned but haven't been published yet.
+
+#### License
+Moved source code and NuGet package from Apache 2.0 license to MIT license. This is more consistent with the other Azure SDKs as well as other open source projects from Microsoft such as DotNet.
```

Comment:
"DotNet" is not normally spelled out that way.  The product is ".NET".  They use "dotnet" as the user name under which they keep their GitHub repos, and in other places where the leading "." isn't allowed... but this seems to refer to the product...?